Makefile

```diff
@@ -94,8 +94,18 @@ lava-boards:
 	lavacli -i $(LAVA_IDENTITY) devices dict set docker-02 devices/docker-generic.jinja2
 	lavacli -i $(LAVA_IDENTITY) devices tags add docker-02 inet
 
-	# Start with creating "virtual devices", which work in any setup,
-	# without additional hardware.
+	# Now create LAVA device types/device for real hardware boards. Note
+	# that these require per actual device configuration files for specific
+	# boards, which are thus different from a developer to developer, and
```

Comment:
```suggestion
	# that these require, per actual device, configuration files 
	# which are different from a developer to developer, and
```

Reply:
'for specific boards' is somewhat redundant with 'per actual device'

Reply:
Thanks Erwan, it looks like "per" here is a stray word here, what I wanted to write is "these require actual device configuration files for specific boards". Hopefully, that sounds ok.

Reply:
Also killed dup "thus", and other minor typo fixes.